src/field/index.js

```diff
@@ -0,0 +1,96 @@
+import {reduce} from 'lodash';
+import {
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLID,
+  GraphQLObjectType,
+  GraphQLSchema,
+  GraphQLScalarType
+} from 'graphql';
+import {getModels} from './../model';
+import {getTypes, nodeInterface} from './../type';
+import {
+  getIdFetcher,
+  getOneResolver,
+  getListResolver
+} from './../query';
+
+function getField(graffitiModel, type) {
+  const {name} = type;
+  const singularName = name.toLowerCase();
+  const pluralName = `${name.toLowerCase()}s`;
+
+  return {
+    [singularName]: {
+      type: type,
+      args: {
+        id: {
+          type: new GraphQLNonNull(GraphQLID),
+          description: `The ID of a ${name}`
+        }
+      },
+      resolve: getOneResolver(graffitiModel)
+    },
+    [pluralName]: {
+      type: new GraphQLList(type),
+      args: reduce(type._typeConfig.fields(), (args, field) => {
```

Comment:
I don't think relay likes root fields with `args` any other than `id`s. See https://github.com/facebook/relay/issues/112 for a discussion.

```
-- Relay Transform Error -- users.jsx --

Error: Invalid root field `users`; Relay only supports root fields with zero or one argument
```